### PR TITLE
Add compression for memcached

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: keyv/memcached/Plugin.h [hunk 1]
```diff
@@ -19,15 +19,19 @@
...
 #include <unordered_map>
+#include <utility>
 
 namespace keyv
 {
 namespace memcached
 {
 namespace
 {
+const std::string COMPRESSOR( "pression::data::CompressorZSTD1" );
```
Comment (tribal-tec) [inline keyv/memcached/Plugin.h]: rm

File: keyv/memcached/Plugin.h [hunk 2]
```diff
@@ -97,49 +102,104 @@ class Plugin : public detail::Plugin
...
+
+        for( const auto& result : results )
+        {
+            // chunk size
+            *reinterpret_cast< uint64_t* >( ptr ) = result.getSize();
+            ptr += sizeof( uint64_t );
+            // chunk
+            ::memcpy( ptr, result.getData(), result.getSize( ));
+            ptr += result.getSize();
+        }
```
Comment (tribal-tec) [inline keyv/memcached/Plugin.h]: private func for compression code

Comment (eile) [inline keyv/memcached/Plugin.h]: done

**Comment by eile** [comment]
> Changelog, version bump missing
done

**Pull request merged**